modules/alloy/src/main/ts/ephox/alloy/api/system/NoContextApi.ts

```diff
@@ -5,27 +5,33 @@ import { AlloyComponent } from '../component/ComponentApi';
 import { AlloySystemApi } from './SystemApi';
 
 const NoContextApi = (getComp?: () => AlloyComponent): AlloySystemApi => {
-  const fail = (event: string) => () => {
-    throw new Error('The component must be in a context to send: ' + event +
-        (getComp ? '\n' + AlloyLogger.element(getComp().element) + ' is not in context.' : '')
-    );
+  const getMessage = (event: string) => `The component must be in a context to execute: ${event}` +
```

Comment:
This was intentionally changed from `send` -> `execute` as send was a little confusing.